codegen/src/main/kotlin/software/amazon/smithy/rust/codegen/rustlang/CargoDependency.kt

```diff
@@ -201,8 +201,10 @@ data class CargoDependency(
         fun SmithyHttp(runtimeConfig: RuntimeConfig) = runtimeConfig.runtimeCrate("http")
         fun SmithyHttpTower(runtimeConfig: RuntimeConfig) = runtimeConfig.runtimeCrate("http-tower")
 
-        fun ProtocolTestHelpers(runtimeConfig: RuntimeConfig) = CargoDependency(
-            "protocol-test-helpers", runtimeConfig.runtimeCrateLocation.crateLocation(), scope = DependencyScope.Dev
+        fun SmithyProtocolTestHelpers(runtimeConfig: RuntimeConfig) = CargoDependency(
+            "smithy-protocol-test-helpers",
```

Comment:
`runtimeConfig.runtimeCrate(...)`. This runs through machinery that can use a different prefix for the runtime crates (eg. if you wanted to codegen and host your own runtime crates)